plugin/federation/protos/proto_gen.sh

```diff
@@ -1,8 +1,8 @@
 protoc -I. \
--I$GOPATH/src/github.com/grpc-ecosystem/grpc-gateway \
```

Comment:
I think the changes here should be reverted.

Reply:
ok！